digits/utils/image.py

```diff
@@ -11,6 +11,20 @@
 except ImportError:
     from StringIO import StringIO
 
+try:
```

Comment:
don't you want to add `pydicom` to `requirements.txt`? If the package is not installed, DIGITS will silently refuse to read DICOM files, which could be confusing to the user.